client/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "clubs-2020",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:8080",
```

Comment:
ive never done this before! Does this mean that our react app runs on port 8080 or that our react app on port 3000 will be able to perform fetch requests to port 8080?

Reply:
yes! all the fetch calls sent by the frontend (listening on localhost:3000) will be directed to the backend server (listening on localhost:8080)